### PR TITLE
Added optional property 'showAllItems' that can enable showing of all items before the user types any text.

### PR DESCRIPTION
**Pull request by ashleydavis**

### CODE REVIEW

File: src/fuzzy-picker.js [hunk 1]
```diff
@@ -130,7 +138,7 @@ export default class FuzzyPicker extends React.Component {
               {this.props.label}
             </span>
             <span className="fuzzy-picker-instructions">
-              <span><strong>tab</strong> or <strong>↑↓</strong> to navigate</span>
+              <span><strong>tab</strong> or <strong>??</strong> to navigate</span>
```
Comment (1egoman) [inline src/fuzzy-picker.js]: I think it'd be desirable to keep the up and down arrows for navigation? I might be misreading the diff, let me know if I'm off base.

**Comment by 1egoman** [comment]
Otherwise, LGTM

File: src/fuzzy-picker.js (hunk 1 above)
Comment (ashleydavis) [reply src/fuzzy-picker.js]: Oh sorry. Not sure what happened there! My text editor must have it's own ideas on that!

**Comment by ashleydavis** [comment]
The PR is updated, reversing the unintentional change with commit 1a911356921ef0ab770af83d0fe9d722f26e923e

**Pull request merged**